### DISCUSSION TITLE
My experience with objeck-lsp

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#404] [General]
It's very useless. It's unable to do code completion. It's unable to make suggestions when you type the dot, for example: `use System.IO.`. It's unable to make suggestions when you type `->`, for example: `Directory->`. Most of the time, I found what was really working was Sublime Text's word-based completion. It's able to show the method's signature by mouse-hovering over the methods, so you could know what type of arguments to feed into the methods. But this is again very useless. You have to open the API document in the browser anyway. How could you know the bundle names and method names before hand? It can't make bundle names and method names suggestions for you! At best, objeck-lsp can only be used as a code linter. But most of the time, the errors suggested by it are simply wrong.

**Comment by objeck** [comment]
Thanks for the feedback. I will look into this.

**Reply by objeck** [reply]
Please send me a copy of your tested code so I may debug it.

**Reply by ghost** [reply]
All code is this way. What would you expect me to send?
